quartz/components/styles/explorer.scss

```diff
@@ -59,7 +77,7 @@ button#explorer {
   }
 
   &.collapsed > .overflow::after {
-    opacity: 0;
+    display: none;
```

Comment:
is this strictly necessary? we originally did opacity to allow for animation as display is a non-animatable property